opm/autodiff/BlackoilModelBase_impl.hpp

```diff
@@ -1800,6 +1800,47 @@ namespace detail {
             }
         }
 
+        /// \brief Compute the L-infinity norm of a vector
+        /// \warn This function is not suitable to compute on the well equations.
+        /// \param a The container to compute the infinity norm on.
+        ///          It has to have one entry for each cell.
+        /// \param info In a parallel this holds the information about the data distribution.
+        template <class Iterator>
```

Comment:
RandomAccessIterator is required below, since that is a very long name it's ok to call it Iterator, but then there should be a `\tparam` doc for it.